deploy-service/common/src/main/java/com/pinterest/deployservice/scm/GithubManager.java

```diff
@@ -108,17 +108,24 @@ public String getType() {
 
     @Override
     public CommitBean getCommit(String repo, String sha) throws Exception {
-        HTTPClient httpClient = new HTTPClient();
-        String url = String.format("%s/repos/%s/commits/%s", apiPrefix, repo, sha);
-
-        // TODO: Do not RETRY since it will timeout the thrift caller, need to revisit
-        String jsonPayload = httpClient.get(url, null, null, headers, 1);
-        GsonBuilder builder = new GsonBuilder();
-        Map<String, Object>
-            jsonMap =
-            builder.create().fromJson(jsonPayload, new TypeToken<HashMap<String, Object>>() {
-            }.getType());
-        return toCommitBean(jsonMap, repo);
+        try
+        {
```

Comment:
Move Up opening bracket {
Java Convention use this style 
https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#472